src/docbuilder/chroot_builder.rs

```diff
@@ -237,6 +237,19 @@ impl DocBuilder {
                 .join("doc");
             let _ = remove_dir_all(crate_doc_path);
         }
+
+        if let Ok(fs) = System::new().mount_at(".") {
```

Comment:
I remember having trouble with this on Crater when `.` wasn't a mount point and I had [to do this](https://github.com/rust-lang/crater/blob/c0587168029236cce79847ad0819b76049004963/src/runner/worker.rs#L203-L220). Can you double check everything works?